### ISSUE TITLE
when training coco dataset, meet not enough memory

### ISSUE THREAD

**Issue by pby5** [facebookresearch/deepmask#31]
Training DeepMask Model	
-- ignore option rundir	
-- ignore option dm	
-- ignore option reload	
-- ignore option gpu	
-- ignore option datadir	
| running in directory /home/gavinpan/workspace/deepmask/exps/deepmask/exp	
| number of paramaters trunk: 15198016	
| number of paramaters mask branch: 1608768	
| number of paramaters score branch: 526337	
| number of paramaters total: 17333121	
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
FATAL THREAD PANIC: (pcall) not enough memory
/home/gavinpan/torch/install/bin/luajit: ...gavinpan/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] not enough memory
stack traceback:
	[C]: in function 'error'
	...gavinpan/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	...gavinpan/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	...gavinpan/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	...gavinpan/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	/home/gavinpan/workspace/deepmask/DataLoader.lua:40: in function '__init'
	/home/gavinpan/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/gavinpan/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataLoader'
	/home/gavinpan/workspace/deepmask/DataLoader.lua:21: in function 'create'
	train.lua:125: in main chunk
	[C]: in function 'dofile'
	...npan/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00405ea0

**Comment by PavlosMelissinos** [comment]
Duplicate of #14

**Comment by poliveirap** [comment]
Thanks, @PavlosMelissinos .
